plugin/ensime.vim

```diff
@@ -19,11 +19,11 @@ endif
 
 augroup ensime
     autocmd!
-    autocmd VimLeave *.scala call ensime#au_vim_leave(expand("<afile>"))
-    autocmd VimEnter *.scala call ensime#au_vim_enter(expand("<afile>"))
-    autocmd BufLeave *.scala call ensime#au_buf_leave(expand("<afile>"))
-    autocmd CursorHold *.scala call ensime#au_cursor_hold(expand("<afile>"))
-    autocmd CursorMoved *.scala call ensime#au_cursor_moved(expand("<afile>"))
+    autocmd VimLeave *.java,*.scala call ensime#au_vim_leave(expand("<afile>"))
+    autocmd VimEnter *.java,*.scala call ensime#au_vim_enter(expand("<afile>"))
+    autocmd BufLeave *.java,*.scala call ensime#au_buf_leave(expand("<afile>"))
+    autocmd CursorHold *.java,*.scala call ensime#au_cursor_hold(expand("<afile>"))
+    autocmd CursorMoved *.java,*.scala call ensime#au_cursor_moved(expand("<afile>"))
```

Comment:
oh, you're right, totally forgot that. TBH it's a PITA, it would be nice to have at least a test that verifies both vim plugin and nvim remote plugin define the same commands and autocommands.